### ISSUE TITLE
command line terratorch predict doesn't create the tif files on Kaggle

### ISSUE THREAD

**Issue by diarray-hub** [IBM/terratorch#814]
Hey all,

I don't know if it's specific to kaggle or if it's an issue that should be fixed with the CustomWriter class but I have been running terratorch cli prediction command on kaggle multiple times and although I'm sure the inference is done properly without any issue. I see no file in the output directory after and no sign/logging showing that it even attempted creating the files. 

I'm running inference wth:

`!terratorch predict -c config.yaml \
  --ckpt_path save_dir/checkpoints/best-epoch=00.ckpt \
  --predict_output_dir predictions/ \
  --data.init_args.predict_data_root /kaggle/input/hack-the-carbon/test/chips/ \
  --data.init_args.predict_dataset_bands [BLUE,GREEN,RED,NIR_NARROW,SWIR_1,SWIR_2] \
  --data.init_args.expand_temporal_dimension true`

And as I said the logging is not actually showing any sign that files are being wriiten, it looks like it just does the inference. Do I need to store that into a varibale somehow and write them myself or am I forgetting a parameter something like that. This is what the above command prints

> 2025-08-20 15:52:54.345015: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:477] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1755705174.369871     227 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1755705174.377503     227 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
INFO: Seed set to 42
/usr/local/lib/python3.11/dist-packages/lightning/pytorch/cli.py:683: `PixelwiseRegressionTask.configure_optimizers` will be overridden by `MyLightningCLI.configure_optimizers`.
INFO: Using bfloat16 Automatic Mixed Precision (AMP)
INFO: You have turned on `Trainer(detect_anomaly=True)`. This will significantly slow down compute speed and is recommended only for model debugging.
INFO: GPU available: True (cuda), used: True
INFO: TPU available: False, using: 0 TPU cores
INFO: HPU available: False, using: 0 HPUs
INFO: Restoring states from the checkpoint path at save_dir/checkpoints/best-epoch=00.ckpt
INFO: LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1]
INFO: Loaded model weights from the checkpoint at save_dir/checkpoints/best-epoch=00.ckpt
Checking stackability.
Predicting ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 237/237 0:04:52 • 0:00:00 0.80it/s it/s it/s 

I really need help figuring this out so if anybody sees this, please tell me it's not one of those things that only happen to me :(

**Comment by Joao-L-S-Almeida** [comment]
Hi, @diarray-hub. I don't got the same error when trying in my usual environment and I still couldn't reproduce your issue in Kaggle, but, before it, I'd like you say me if there is any special log when running it with the branch `debug/prediction`. 
By the way, is it possible to share your config ? Maybe there is something we could change in the config.  Thanks.

**Comment by romeokienzler** [comment]
@diarray-hub closing due to inactivity, feel free to reopen
